src/convertToObject.js

```diff
@@ -6,7 +6,23 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styles = sourceString
+    .trim()
+    .split(';')
+    .filter((style) => style.trim() !== '');
```

Comment:
The chaining of methods here is correct, but the indentation should be consistent to improve readability. Each method call should be on a new line with the same indentation level.